### PR TITLE
Add pyvcf to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
License file was added to the feedstock and an upstream PR was created to fix this issue.

cf: jamescasbon/PyVCF#295

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/bioconda-recipes, please advise how to proceed with pyvcf package. 

The bioconda community prefers for the package to stay in bioconda, and I need to package `apache-beam` and others for airflow which has a install-requirement on pyvcf.

**Comment by sodre** [comment]
@bgruening, this is ready to merge.

File: recipes/pyvcf/LICENSE [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (marcelotrevisani) [inline recipes/pyvcf/LICENSE]: Please, remove this file. This is not necessary since the ``LICENSE`` file is already in the repository and being package

Comment (sodre) [reply recipes/pyvcf/LICENSE]: It is not. Please see my initial post with a reference to the PR for changing the MANIFEST.in file.

Until then the license file should stay here.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**